### PR TITLE
Conformance results for v1.20/inspur-iop-amd

### PR DESCRIPTION
**Pull request by jwz-ecust**
Signed-off-by: zhangjiawei01 <zhangjiawei01@inspur.com>

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [ ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [ ] Did you include the product/project logo in SVG, EPS or AI format?  
* [ ] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [ ] If your product/project is open source, did you include the `repo_url`?
* [ ] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

**Comment by jwz-ecust** [comment]
@ErinThacker   Could you please check out this pr? Thanks!

**Comment by hh** [comment]
I asked the @cncf-ci bot to remove all labels.

This should allow the verifications to run again.

**Comment by hh** [comment]
The specific tests that must be passed for 1.20 conformance listed in a couple places:
- https://github.com/cncf/k8s-conformance/blob/master/docs/KubeConformance-1.20.md#list-of-tests
-  https://github.com/kubernetes/kubernetes/blob/v1.20.0/test/conformance/testdata/conformance.yaml

The easiest way to run the tests you want is by running sonobuoy.

https://github.com/cncf/k8s-conformance/blob/master/instructions.md#running

This pulls down the correct version of the tests to run for your cluster:
https://github.com/vmware-tanzu/sonobuoy/blob/eed1cd73c1eb583d0a983966de78be47c14b3484/cmd/sonobuoy/app/gen.go#L137-L140

Currently the uploaded logs appear to be missing the passing results for at least the following tests:
- [sig-network] Services should be able to create a functioning NodePort service [Conformance]
- [k8s.io] Pods should allow activeDeadlineSeconds to be updated [NodeConformance] [Conformance]

@bernokl can you verify the process is functioning as designed?

**Comment by hh** [comment]
I noted that there are multiple submissions here (amd64 + arm64).

The automation currently only works for one folder / product at a time.

Please resubmit as separate PRs for each arch.

**Comment by jwz-ecust** [comment]
> I noted that there are multiple submissions here (amd64 + arm64).
> 
> The automation currently only works for one folder / product at a time.
> 
> Please resubmit as separate PRs for each arch.


ok, Thanks!

File: v1.20/inspur-iop-arm64/id_rsa-arm [hunk 1]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (hh) [inline v1.20/inspur-iop-arm64/id_rsa-arm:1]: Might want to verify where you use this private key.
Probably don't want to upload this.

**Pull request closed without merging**